.github/workflows/package-on-fork.yaml

```diff
@@ -0,0 +1,127 @@
+          PACKAGE_CREATION_PATH: "${{ steps.validateAndCreatePackage.outputs.packageCreationPath }}"
+          DATA_FOLDER_PATH: "${{ steps.validateAndCreatePackage.outputs.dataFolderPath }}"
+          DATA_INPUT_FILE_NAME: "${{ steps.validateAndCreatePackage.outputs.dataInputFileName }}"
+          SOLUTION_NAME: "${{ steps.validateAndCreatePackage.outputs.solutionName }}"
+          SOLUTION_SUPPORTED_BY: "${{ steps.validateAndCreatePackage.outputs.solutionSupportedBy }}"
+          RUNID: "${{ github.run_id }}" 
+          PULL_REQUEST_NUMBER: "${{ github.event.pull_request.number }}"
+          IS_CREATE_PACKAGE: ${{ steps.validateAndCreatePackage.outputs.isCreatePackage }}
+        if: ${{ success() && (steps.validateAndCreatePackage.outcome == 'success' && env.IS_CREATE_PACKAGE && env.PACKAGE_CREATION_PATH != '' && env.BLOBNAME != '') }}
+        with: 
+          name: "${{ env.BLOBNAME }}"
+          path: "${{ env.PACKAGE_CREATION_PATH }}"
+        
+      - name: Upload Data File Artifacts
+        id: uploadDataFileArtifact
+        uses: actions/upload-artifact@e0057a5b76f2fdad976135e8dd7b691e632b9056
+        env:
+          DATA_FOLDER_PATH: "${{ steps.validateAndCreatePackage.outputs.dataFolderPath }}"
+          DATA_INPUT_FILE_NAME: "${{ steps.validateAndCreatePackage.outputs.dataInputFileName }}"
+        if: ${{ success() && (steps.validateAndCreatePackage.outcome == 'success' && env.DATA_FOLDER_PATH != '' && env.DATA_INPUT_FILE_NAME != '') }}
+        with: 
+          name: "${{ env.DATA_INPUT_FILE_NAME }}"
+          path: "${{ env.DATA_FOLDER_PATH }}"
+
+      - name: create-new-pr
+        id: create-new-pr
+        env:
+          SOLUTION_NAME: "${{ steps.validateAndCreatePackage.outputs.solutionName }}"
+          PULL_REQUEST_NUMBER: "${{ github.event.pull_request.number }}"
+          CURRENT_BRANCH_NAME: "${{ github.event.pull_request.head.ref }}"
+          ASSIGNEES: "${{ github.actor }}"
+        if: ${{ success() && (steps.validateAndCreatePackage.outputs.isCreatePackage && steps.validateAndCreatePackage.outputs.packageCreationPath != '' && steps.validateAndCreatePackage.outputs.blobName != '') }}
+        uses: peter-evans/create-pull-request@5b4a9f6a9e2af26e5f02351490b90d01eb8ec1e5
+        with:
+          committer: GitHub <noreply@github.com>
+          assignees: "${{ env.ASSIGNEES }}"
+          signoff: false
+          branch: "${{ env.CURRENT_BRANCH_NAME }}-automated-pr"
+          base: master
+          delete-branch: false
+          title: '[GitHub Bot] 🤖 Package Created For ${{ env.SOLUTION_NAME }} Solution'
+          body: |
+            Automation have successfully generated package for solution '${{ env.SOLUTION_NAME }}' based on Fork Pull request #${{ env.PULL_REQUEST_NUMBER }}.
+          labels: |
+            auto-package
+            Solution
+
+      - uses: fixpoint/azblob-upload-artifact@41eaaaa8fa50a5ba144d1d106c0e4ca11ca60c08
```

Comment:
Please removed this. not required for now

Reply:
done